package.json

```diff
@@ -52,11 +55,11 @@
   "dependencies": {
     "@azure/msal-browser": "^3.20.0",
     "csv-string": "^4.1.1",
-    "date-fns": "^2.30.0",
+    "date-fns": "^4.1.0",
```

Comment:
Note: this change of date-fns from v2 to v4 might be a breaking change for our components, we should probably include this update in the next major release